index.html

```diff
@@ -18,17 +18,24 @@
         <div class="header">
           <div class="logo"><a href="#">My Logo</a></div>
           <nav>
+            <div class="menu-content">
+              <ul>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+              </ul>
+            </div>
             <span class="bars"><i class="fa-solid fa-bars"></i></span>
           </nav>
         </div>
       </header>
 
       <section class="intro">
```

Comment:
- [ ] Please reduce the gap between social media icons and make sure it matches the design on the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/160193077-7fd59172-9995-4cec-a784-f3767d2025fa.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/160193117-c37cf6d7-3498-4079-be17-c37fafa16517.png)